lib/devbox_launcher/models/box.rb

```diff
@@ -38,6 +38,7 @@ def start_cmd
       args = {
         project: config[:project],
         account: account,
+        zone: (config[:zone] || 'us-central1-a'),
       }.map do |(key, val)|
         ["--#{key}", val].join("=")
       end.join(" ")
```

Comment:
We should exclude blank values though, in case the zone isn't specified in the config
```suggestion
      }.each_with_object([]) do |(key, val), arr|
        next if val.blank?
        arr << ["--#{key}", val].join("=")
      end.join(" ")
```